### ISSUE TITLE
Agda could fail to build if a .agda-lib file exists in a parent directory

### ISSUE THREAD

**Issue by nad** [agda/agda#3648]
If you put a `.agda-lib` file, with a dependency on a library that is not listed in the default `libraries` file, in a directory above the root directory of the Agda source tree, then Agda fails to build:
```
$ echo "depend: non-existing-library" > ../.agda-lib
$ make install
[...]
Generating Agda library interface files...
... [...]/src/data/lib/prim/Agda/Builtin/Bool.agda
 Library 'non-existing-library' not found.
 Add the path to its .agda-lib file to
   '[...]/.agda/libraries'
 to install.
[...]
```
I encountered this problem when using `agda-bisect`. I suppose it could also affect `cabal install Agda`. I suggest that we use `--no-libraries` when typechecking the builtin files.

**Comment by nad** [comment]
I'm not sure if this is a regression. @asr, do you know? If it is, then we should perhaps fix this before releasing Agda 2.6.0.

**Comment by asr** [comment]
In previous versions, Agda generated a warning instead of an error if there was a problem when typechecking the builtin files. I changed the warning by an error due to some issues (e.g. #3128). I'm not sure either if the issue in the OP is a regression.

**Comment by nad** [comment]
@asr, I leave it to you to decide whether to include this fix in 2.6.0.

**Comment by asr** [comment]
I'm planning to merge `master` in `release-2.6.0`, so this fix will be included.
